snapshots/src/archive.rs

```diff
@@ -179,7 +179,7 @@ pub fn archive_snapshot(
         ("archive_format", archive_format.to_string(), String),
         ("duration_ms", timer.as_ms(), i64),
         (
-            if snapshot_kind.is_full_snapshot() {
+            if archive_kind.is_full_archive() {
```

Comment:
We can do:

```rust
if snapshot_archive_kind == SnapshotArchiveKind::Full {
[..]
```

And then we don't need to create the `is_full_archive()` method on `SnapshotArchiveKind`.